README.md

```diff
@@ -1 +1,151 @@
+(You need to approve enough tokens to rfq contract before creating an exchange request)
+
+``` shell
+curl -X POST "https://code.flows.network/webhook/ekbbxC47MjjtIaP8RmO8/submit-request?token-out=0x30D30c71d8618Ce42783eDd2C7Ae6f15eeD69Fec&token-in=0x948Fa9010EFBEed5f4943893a383B7e2210bA145&amount=100" \
+-d '{"PRIVATE_KEY": "Exchange request owner private key"}'
+```
+Then you can see the transaction result.<br>
+
+### List requests
+
+After creating an exchange request, use the `list-requests` function to query all the exchange requests.<br>
+Copy and paste the endpoint URL to your browser and add `/list-requests`.
+Then you can see all the exchange requests in the rfq-marketplace.
+
+<img width="964" alt="image" src="https://i.imgur.com/4wJWy9n.png">
+
+### Response exchange
+
+If you want to exchange tokens with others, use `submit-response` to respond to other's exchange requests.<br>
+You can copy and paste the endpoint URL to your shell and add `/submit-response?request-id=0&amount=10&lifetime=3600`. <br>
+`request-id` : The request you want to exchange. <br>
+`amount` : The amount you want to exchange out ([token-in](#create-an-exchange-request)).<br>
+`lifetime` : This response lifetime unit in seconds. You can withdraw after the lifetime.<br>
+
+(You need to approve enough tokens to rfq contract before the response exchange)
+
+``` shell
+curl -X POST "https://code.flows.network/webhook/ekbbxC47MjjtIaP8RmO8/submit-response?request-id=0&amount=10&lifetime=3600" \
+-d '{"PRIVATE_KEY": "Buyer private key"}'
+```
+Then you can see the transaction result.<br>
+
+### Get request
+
+You can use `get-request` to query all the buyers of the exchange request. <br>
+Copy and paste the endpoint URL to your browser and add `/get-request?request-id=0`.
+Then you can see the buyer information.
+
+The amount the buyer wants to exchange is the amount of token-in to you.
+
+<img width="964" alt="image" src="https://i.imgur.com/r780Jn6.png">
+
+### Accept exchange
+
+If you are the owner of the exchange request, you have the right to decide which buyer you want to exchange.
+> You need accept not expired response.
+
+Using `accept-exchange` to accept the response.
+You can copy and paste the endpoint URL to your shell and add `/accept-exchange?request-id=0&response-id=0`.
+
+``` shell
+curl -X POST "https://code.flows.network/webhook/ekbbxC47MjjtIaP8RmO8/accept-exchange?request-id=0&response-id=0" \
+-d '{"PRIVATE_KEY": "Exchange request owner private key"}'
+```
+Then you can see the transaction result.<br>
+
+### Withdraw
+
+When the request is not finished until exceeds `requestLiveTime` or the owner accepts the exchange, all the buyer and owner can use `withdraw` to withdraw the token that is locked in rfq-marketplace. You can copy and paste the endpoint URL to your shell and add `/withdraw?request-id=0` <br>
```

Comment:
It seems that the `requestLiveTime` is set for the entire marketplace contract? Maybe we should allow the requester to set it for his own request (with a default if he does not set it)? Each response could have its own live time now after all. Thanks.

Reply:
I have added it. Please check, thank you.